CHANGES.md

```diff
@@ -6,6 +6,8 @@
 - [FIXED] Ensure IAM API key can be correctly changed.
 - [FIXED] Callback with an error when a user cannot be authenticated using IAM.
 - [FIXED] Ensure authorization tokens are not unnecessarily requested.
+- [IMPROVED] Disable cookie authentication if credentials are missing and the
```

Comment:
How about `Do not apply cookie authentication by default in the case that no credentials are provided.`